examples/basics/visuals/colorbar_visual.py

```diff
@@ -81,7 +83,7 @@ def get_mandlebrot_escape_values(width, height):
     return v_get_num_escape_turns(*grid).astype(np.float)
 
 
-def get_vertical_bar():
+def get_vertical_bar(pos, halfdim):
```

Comment:
that doesn't particularly make sense to me. How would you auto-detect orientation in that case? `(minor_length, major_length)` doesn't tell you _which_ axis is minor and which is major